test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -71,6 +72,8 @@ public class EnvironmentHelper
 
         public bool DebugModeEnabled { get; set; }
 
+        public Dictionary<string, string> ExtraEnvironmentVariables { get; set; } = new Dictionary<string, string>();
```

Comment:
"Extra" seems too vague, especially since this is a public property in a public class. I'm not sure of an alternative yet.

Reply:
I did a small refactoring and changed this to `CustomEnvironmentVariables`. Is that any better?

Reply:
Yeah, I like that better. Thanks for changing it Zach!